### ISSUE TITLE
504 gateway timeout

### ISSUE THREAD

**Issue by derhuerst** [alanshaw/david-www#351]
```shell
curl -L 'https://david-dm.org/derhuerst/time-tracking'
```

```
<html>
<head><title>504 Gateway Time-out</title></head>
<body bgcolor="white">
<center><h1>504 Gateway Time-out</h1></center>
<hr><center>nginx/1.4.6 (Ubuntu)</center>
</body>
</html>
```

**Comment by derhuerst** [comment]
Generally, David seems to be extremely slow, both for `/` and for individual repos.

**Comment by alanshaw** [comment]
Looks likely that the recent slowness and disruption to service was caused by the [memory leak](https://github.com/nodejs/node/commit/7a59449478) in Node.js 6.7. This has now been resolved with an upgrade to 6.8.
